app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -90,16 +90,22 @@ module.exports = createReactClass
     @menus = []
 
   componentDidMount: ->
-    annotationValues = @props.annotation.value
-    unless annotationValues.length
-      @props.annotation.value = @props.task.selects.map -> {value: null, option: false}
+    @setDefaultValues()
 
     if @props.autoFocus is true
       @handleFocus()
 
   componentDidUpdate: (prevProps) ->
-    if prevProps.task isnt @props.task and @props.autoFocus is true
-      @handleFocus()
+    if prevProps.task isnt @props.task
+      @setDefaultValues()
+      if @props.autoFocus is true
+        @handleFocus()
+
+  setDefaultValues: ->
+    unless @props.annotation.value.length
+      value = @props.task.selects.map () -> { option: false, value: null }
+      newAnnotation = Object.assign @props.annotation, {value}
```

Comment:
There's a missing {} here. This modifies `@props.annotation`, rather than copying it and creating a new annotation.

`newAnnotation = Object.assign {}, @props.annotation, {value}